src/ZipkinBundle/Components/Messenger/ZipkinSendHandler.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+
+namespace ZipkinBundle\Components\Messenger;
+
+use Symfony\Component\Messenger\Envelope;
+
+use Zipkin\Kind;
+use Zipkin\Propagation\B3;
+use Zipkin\Tags;
+use Zipkin\Tracing;
+
+class ZipkinSendHandler implements ZipkinHandlerInterface
+{
+    /**
+     * @var \Zipkin\Tracer
+     */
+    private $tracer;
+    /**
+     * @var B3
+     */
+    private $b3;
+
+    public function __construct(Tracing $tracing, B3 $b3)
```

Comment:
You don't need to pass `$b3` in here as `$tracing->getPropagation()` is going to give it to you :).

Reply:
Done